pkg/neotest/basic.go

```diff
@@ -297,13 +297,32 @@ func AddSystemFee(bc *core.Blockchain, tx *transaction.Transaction, sysFee int64
 }
 
 // AddNetworkFee adds network fee to the transaction.
-func AddNetworkFee(bc *core.Blockchain, tx *transaction.Transaction, signers ...Signer) {
+func AddNetworkFee(t testing.TB, bc *core.Blockchain, tx *transaction.Transaction, signers ...Signer) {
 	baseFee := bc.GetBaseExecFee()
 	size := io.GetVarSize(tx)
 	for _, sgr := range signers {
-		netFee, sizeDelta := fee.Calculate(baseFee, sgr.Script())
-		tx.NetworkFee += netFee
-		size += sizeDelta
+		if csgr, ok := sgr.(ContractSigner); ok {
+			sc, err := csgr.InvocationScript(tx)
+			require.NoError(t, err)
+
+			ic, err := bc.GetTestVM(trigger.Verification, nil, nil)
+			require.NoError(t, err)
+
+			ic.UseSigners(tx.Signers)
+			ic.VM.GasLimit = -1
+			ic.SpawnVM()
+
+			bc.InitVerificationContext(ic, csgr.ScriptHash(), &transaction.Witness{InvocationScript: sc, VerificationScript: []byte{}})
+
+			require.NoError(t, ic.VM.Run())
+
+			tx.NetworkFee += ic.VM.GasConsumed()
+			size += io.GetVarSize(sc) + 1 // empty verification script size
```

Comment:
s/`1`/`io.GetVarSize(csgr.Script())`? The same meaning, but then you don't need a comment.

Reply:
Fixed